nerdamer.core.js

```diff
@@ -10042,14 +10042,15 @@ var nerdamer = (function (imports) {
         text: function () {
             return 'matrix(' + this.elements.toString('') + ')';
         },
-        latex: function (option) {
+        latex: function (type = 'v', options) {
```

Comment:
Please make type the second argument. Making it the first argument will break existing code. Also can you use `type = type || 'v'` or something similar for default options so the code doesn't break in older browsers?